build/tasks/install_jsdom.js

```diff
@@ -0,0 +1,29 @@
+module.exports = function( grunt ) {
+	grunt.registerTask( "jsdom", function() {
+		var done = this.async(),
+			pkg = grunt.config( "pkg" ),
+			version = pkg.jsdomVersions[
+
+				// Unfortunately, this is currently the only
+				// way to tell the difference between Node and iojs
+				/^v0/.test( process.version ) ? "node" : "iojs"
+			];
+
+		// Link npm so we can use the global one
+		grunt.util.spawn( {
+			cmd: "npm",
+			args: [ "link", "npm" ]
+		}, function() {
+			var npm = require( "npm" );
+			npm.load(function() {
+				npm.commands.ls( [ "jsdom" ], function( data ) {
```

Comment:
How fast is it? `npm ls` is slow enough that I built [check-dependencies](http://npmjs.com/package/check-dependencies) to have a quick check for installed deps. Manually doing `require('./node_modules/jsdom/package.json').version` (+ checking if the file exists first) would be faster, less code & I doubt it'll ever stop working.